modules/govuk_envsys/manifests/init.pp

```diff
@@ -4,8 +4,10 @@
 # LC_ALL.
 #
 class govuk_envsys {
+  $development = $::domain == 'development'
+
   file { '/etc/environment':
-    ensure => present,
-    source => 'puppet:///modules/govuk_envsys/etc/environment',
+    ensure  => present,
+    content => template('govuk_envsys/environment.erb'),
```

Comment:
I wonder if this GOPATH change would be better in a file in `/etc/profile.d` which is only created for dev vm?